cas-offinder.cpp

```diff
@@ -440,19 +430,31 @@ void Cas_OFFinder::compareAll(const char* outfilename) {
 	delete [] cl_compare_flags;
 }
 
-void Cas_OFFinder::init_platforms() {
-	oclGetPlatformIDs(MAX_PLATFORM_NUM, platforms, &platform_cnt);
-	if (platform_cnt == 0) {
-		cerr << "No OpenCL platforms found. Check OpenCL installation!" << endl;
-		exit(1);
+void Cas_OFFinder::releaseLociinfo() {
+	unsigned int dev_index;
+
+	for (dev_index = 0; dev_index < m_activedevnum; dev_index++) {
+		free((void *)m_mmcounts[dev_index]);
+		free((void *)m_flags[dev_index]);
+		free((void *)m_directions[dev_index]);
+		free((void *)m_mmlocis[dev_index]);
 	}
+	m_directions.clear();
+	m_mmlocis.clear();
+	m_mmcounts.clear();
+	m_locicnts.clear();
+	clearbufvec(&m_mmlocibufs);
+	m_flags.clear();
+	clearbufvec(&m_mmcountbufs);
+	clearbufvec(&m_directionbufs);
 }
+
 void Cas_OFFinder::print_usage() {
 	unsigned int i, j;
-	cout << "Cas-OFFinder 3.0.0b (" << __DATE__ << ")" << endl <<
+	cout << "Cas-OFFinder v" << CAS_OFFINDER_VERSION << " (Compiled on: " << __DATE__ << ")" << endl <<
```

Comment:
I consider removing the prefix 'v' from the version number from 3.0 release

Reply:
Sorry I missed this!

I removed the `v` on the `develop-build-changes` changes of #36, and rebased here too.